src/lib/layer.js

```diff
@@ -284,6 +290,16 @@ export default class Layer {
   }
 
   /**
+   * Returns the picking color of the object at 'selectedObjectIndex' prop.
```

Comment:
Do we need this to be a public method on the class or should we make it private (underscore)? If public, we should add it to `Layer` docs.